styles/style.css

```diff
@@ -0,0 +1,150 @@
+h1, h2, h3, #welcome sub {
+  font-family: 'Major Mono Display', monospace;
+}
+
+nav, #welcome a, section:not(.welcome) {
+  font-family: 'Glegoo', serif;
+}
+
+nav ul {
+  display: grid;
+  grid-template: 100% / 60% 20% 20% ;
+  list-style-type: none;
+  padding: 20px;
+}
+
+h1 {
+  font-size: 50px;
+}
+
+#welcome {
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#welcome sub {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 30px;
+  margin-top: 10px;
+  font-size: 30px;
+}
+
+#welcome a {
+  border: solid lightgray 1.5px;
+  border-radius: 10px;
+  text-decoration: none;
+  padding: 13px;
+  color: black;
+  margin: 25px;
+}
+
+#welcome a:hover {
+  background-color: #ecd9c6;
+  font-style: italic;
+}
+
+.quote {
+  height: 200px;
+  display: grid;
+  grid-template: 2fr 1fr / 100%;
+  border-top: solid lightgray 1.5px;
+  border-bottom: solid lightgray 1.5px;
+  font-style: italic;
+  padding: 50px;
+  font-family: 'Frank Ruhl Libre', serif;
+  background-color: #8DADCC;
+}
+
+.quote blockquote {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
```

Comment:
The blockquote wrapped very well when I adjusted the screen size. One improvement that you could make is to display the quotation marks around the quote, using blockquote::before {content: open-quote;} and blockquote::after{content: close-quote;}